src/app/cli/src/app.rs

```diff
@@ -654,6 +730,58 @@ pub fn register_config_in_catalog(
     ));
 }
 
+fn get_app_database_config(
+    config: &config::CLIConfig,
+    multi_tenant_workspace: bool,
+    init_command: bool,
+) -> AppDatabaseConfig {
```

Comment:
Maybe move to `database.rs`, since it concentrates everything about the databases.

Reply:
Done